src/Microsoft.AspNetCore.Antiforgery/AntiforgeryTokenSet.cs

```diff
@@ -15,28 +15,57 @@ public class AntiforgeryTokenSet
         /// </summary>
         /// <param name="requestToken">The token that is supplied in the request.</param>
         /// <param name="cookieToken">The token that is supplied in the request cookie.</param>
-        public AntiforgeryTokenSet(string requestToken, string cookieToken)
+        /// <param name="formFieldName">The name of the form field used for the request token.</param>
+        /// <param name="headerName">The name of the header used for the request token.</param>
```

Comment:
Hmm, may be clearer to use `<paramref name="requestToken"/>` and `<paramref name="cookieToken"/>`.